src/TodoList.js

```diff
@@ -1,5 +1,28 @@
 class TodoList {
+  constructor() {
+    this.todoList = [];
+  }
 
+  displayTodoList() {
+    return this.todoList;
+  }
+
+  create(text) {
+    const todo = {
+      id: this.todoList.length + 1,
+      text: text,
+      status: "incomplete",
+    };
+
+    this.todoList.push(todo);
+  }
+
+  isComplete(status) {
```

Comment:
I'm not sure which requirement is this method for.
Is this returning the incomplete items?